packages/slate-react/src/components/editable.tsx

```diff
@@ -250,10 +250,20 @@ export const Editable = (props: EditableProps) => {
   // The autoFocus TextareaHTMLAttribute doesn't do anything on a div, so it
   // needs to be manually focused.
   useEffect(() => {
-    if (ref.current && autoFocus) {
-      ref.current.focus()
-    }
-  }, [autoFocus])
+      if (ref.current && autoFocus) {
+        if(window.getSelection && document.createRange) {
+          const range = document.createRange();
+          ref.current.focus();
```

Comment:
Is it essential to call `ref.current.focus()` here and not at the end of this `if` clause? Maybe we can move this call out of the `if` statement?